_includes/releases/v23.1/v23.1.0-alpha.9.md

```diff
@@ -0,0 +1,80 @@
+
+<h3 id="v23-1-0-alpha-9-miscellaneous">Miscellaneous</h3>
+
+- Several computed columns have been added to `crdb_internal.statement_statistics_persisted` and `crdb_internal.transaction_statistics_persisted` views and indexed in the system tables referenced by the views:
+
+    - `execution_count`
+    - `service_latency`
+    - `cpu_sql_nanos`
+    - `contention_time`
+    - `total_estimated_execution_time`
+    - `p99_latency`
+
+    [#99417][#99417]
+
+<h3 id="v23-1-0-alpha-9-bug-fixes">Bug fixes</h3>
+
+- Fixed pagination bugs when searching or filtering within the [Databases](ui-databases-page.html) or viewing the details of a database. [#99513][#99513]
+- Fixed a rare bug introduced in v22.2.0 that could cause a node to crash with an `attempting to append refresh spans after the tracked timestamp has moved forward` error when querying the `pg_catalog` and `crdb_internal` virtual tables. If you are experiencing this bug, set the `sql.distsql.use_streamer.enabled` [cluster setting](cluster-settings.html) to `false` before upgrading a cluster to v23.1. [#99443][#99443]
+- Fixed a bug that could erroneously cause multiple garbage-collection jobs to be created when executing a [`DROP SCHEMA ... CASCADE`](drop-schema.html) command, one job for each table and one for the cascaded `DROP` itself. [#99706][#99706]
+- Fixed a bug in the [Schema Insights](ui-schema-insights.html) and [Table Insights](ui-table-insights.html) pages that prevented a recommendation to drop an index from being executed if the index's name contained a space. [#100023][#100023]
+- Fixed a rare bug that prevented the garbage-collection job for a [`TRUNCATE`](truncate.html) command from successfully finishing if the table descriptor had already been garbage-collected. The garbage-collection job now succeeds in this situation. [#100009][#100009]
+- Fixed a rare bug that could cause a query of a virtual table in the [`crdb_internal`](crdb-internal.html) or [`pg_catalog`](pg-catalog.html) system catalog to hang indefinitely if the query returned an error. [#99969][#99969]
+- Fixed a bug introduced prior to v21.2 that could cause the SQL gateway node to crash if you [created a view](create-view.html) with a circular or self-referencing dependencies. This situation no longer crashes the node, and a `cyclic view dependency for relation` error is now logged. [#100159][#100159]
+- Several rare bugs have been fixed that could cause corruption in the existing primary index when a rollback occurs concurrent to adding or removing a [column family](https://www.cockroachlabs.com/docs/stable/column-families.html). This could lead to subsequent unavailability of the table. [#100030][#100030]
+- Fixed a bug in the [Statements Detail](ui-statements-page.html) and [Transactions Details](ui-transactions-page.html) pages that could cause a node to crash with an out-of-memory (OOM) exception. [#99550][#99550]
```

Comment:
I'm not sure if "Statement Details" is still the name of that section of the UI. Tried to look in `cockroach demo` and didn't see it. It would be best to double-check the correct title. It looks like "Statement Fingerprint" now, but I'm only guessing that this is where the bugfix was applied.

```suggestion
- Fixed a bug in [**Statement Details**](ui-statements-page.html) and [**Transaction Details**](ui-transactions-page.html) that could cause a node to crash with an out-of-memory (OOM) exception. [#99550][#99550]
```